README.md

```diff
@@ -21,6 +21,7 @@ This work presents **Video Depth Anything** based on [Depth Anything V2](https:/
 ![teaser](assets/teaser_video_v2.png)
 
 ## News
+- **2025-02-08:** Enable autocast inference. Support grayscale video, NPZ and EXR output formats.
```

Comment:
1. add the detailed information in "Usage" for the argument of run.py.
2. add columns in the "Pretrain models" for the inference time and VRAM of fp32 and fp16.

Reply:
And please change the PR title with more informational descriptions.

Reply:
The table may put the delta1 metric also.